cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -114,16 +121,26 @@ public static void beforeAll() {
 
         vertx = Vertx.vertx();
         sharedWorkerExecutor = vertx.createSharedWorkerExecutor("kubernetes-ops-pool");
+        
+        // Configure Cruise Control mock
+        serverPort = CruiseControl.REST_API_PORT;
```

Comment:
As you are changing this, would it make sense to change it everywhere the same way or not at all?
* Here you create a static variable but assign it a fixed value
* In `KafkaRebalanceStateMachineTest` you keep using `CruiseControl.REST_API_PORT` directly
* In `CruiseControlClientTest` you seem to change it to using static variable and `TestUtils.getFreePort()` => this seems to me like the best variant

If you are alread changing it, it seems like it might make sense to change it the same way everywhere.

Reply:
The problem with those tests is that they don't use the CC client directly, but through `KafkaRebalanceAssemblyOperator` where CC port is hard coded to `CruiseControl.REST_API_PORT`. I didn't changed it to keep the PR focused on tests, but I can do it if you prefer.